web-registry/src/pages/Marketplace/Storefront/hooks/useCancelSellOrderSubmit.tsx

```diff
@@ -71,7 +71,18 @@ const useCancelSellOrderSubmit = ({
 
     const baseDenom = await getDenomtrace({ denom: askDenom });
 
+    const projectId =
+      selectedSellOrder.project?.id ??
+      batchDenom.substring(0, batchDenom.indexOf('-', 4));
```

Comment:
Same.

Reply:
This is not a case like the previous ones. The data should be included in `selectedSellOrder` , but in its definition in the types it leaves room for undefined. So in such a case it is extracted from the batchDenom..